README.md

```diff
@@ -62,7 +62,7 @@ To build the Riak Java Client, you must have the correct version of the riak_pb
 
 ```
 git clone https://github.com/basho/riak_pb
-git checkout java-2.1.1.0
+git checkout 2.1.2.0
```

Comment:
We would need to add a test similar to this one, that adds a context, to make sure it encodes correctly:  https://github.com/basho/riak-java-client/blob/merge-dpclient-ts1.3/src/test/java/com/basho/riak/client/core/codec/TermToBinaryCodecTest.java#L133

I think I added everything else in that branch to add the coverage context to the TTB TS Queries though.